python/tests/integration/core/test_core_example.py

```diff
@@ -0,0 +1,197 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest import TestCase
+
+from hamcrest import *
+# EXAMPLE START MARKER
+from typedb.driver import *
+
+# EXAMPLE END MARKER
+
+
+class TestExample(TestCase):
+
+    def setUp(self):
+        with TypeDB.core_driver(TypeDB.DEFAULT_ADDRESS) as driver:
+            if driver.databases.contains("typedb"):
```

Comment:
"typedb" should be a constant somewhere, since we're reusing it throughout the example IMO